modules/33_CDR/portfolio/declarations.gms

```diff
@@ -22,7 +22,7 @@ s33_OAE_glo_limit           "global limit for OAE [tC / a]"
 parameters
 p33_fedem(all_te,all_enty)               "final energy demand of each technology [EJ/GtC] (for EW the unit is [EJ/Gt stone])"
 p33_LimRock(all_regi)                    "regional share of EW limit [fraction], calculated ex ante for a maximal annual amount of 8 Gt rock in D:\projects\CEMICS\paper_technical\supply_curve_transport_remind_regions.m"
-p33_co2_rem_rate(rlf)                    "carbon removal rate [fraction of annual reduction of total carbon removal potential], multiplied with grade factor"
+p33_co2_rem_rate(rlf)                    "stone weathering rate, exponent in exponential decay function (based on percent action of stone weathering and including climate grade)"
```

Comment:
Thanks for spotting this Tabea! As it seems to be the stone weathering rate rather than the actual removal rate, I would be in favour of renaming the parameter accordingly.
Maybe p33_rockew_rate? What do you think?

Reply:
How about p33_rock_weath_rate?
We should also rename it to s33_rock_weath_fraction then.
Depends on whether we want to shift the focus from the removal to the weathering (as per your other comment)